entries/1/15five.com.json

```diff
@@ -4,7 +4,8 @@
     "tfa": [
       "totp"
     ],
-    "documentation": "https://success.15five.com/hc/en-us/articles/360002698811",
+    "documentation": "https://success.15five.com/hc/en-us/articles/360002698811#h_01G1GXTSKCGMSC3ECKRFSC8NWM",
+    "notes": "You cannot enable two-step authentication if your organization uses SSO.",
```

Comment:
This part isn't something we usually write about since SSO means authentication is handled externally and external authentication providers are out of scope.

Reply:
From their documentation it sounds like they leave authentication up to the organisations. If the organisation chooses to use SSO then they (15five.com) have no ability to enforce the 2FA because you are taking authentication out of their hands. If I, as a user, wanted to turn on 2FA but my organisation turned on SSO then I am out of luck. I feel like this note is pretty important since the user is forced into a (potentially) less secure option because the organisation wants to handle SSO their own way. How would you like me to proceed?

Reply:
As both you and I mentioned, SSO means the organization/SAML-provider handles all forms of authentication.
SSO/SAML means authentication is outside the trust-boundary of the site and instead inside the trust-boundary of the SAML-provider. Thus out of scope for the entry.

Just like we don't label sites offering authentication via Google or Facebook as having 2FA, we don't note exceptions for SSO/SAML.